scripts/tests/test_execute_angle_aware_bilateral_filtering.py

```diff
@@ -18,37 +31,74 @@ def test_help_option(script_runner):
     assert ret.success
 
 
-def test_asym_basis_output(script_runner):
+@pytest.mark.parametrize("out_fodf",
+                         [os.path.join(cwd, "test_fodf.nii.gz")])
+def test_asym_basis_output(script_runner, mock_filtering, out_fodf):
     os.chdir(os.path.expanduser(tmp_dir.name))
```

Comment:
It seems like the problem is from this line.

We switch to a tmp_dir for output and all, but the inputs are not in that folder, they are still in the `get_home()` (~/.scilpy/SOMETHING/)

The tests are not passing because 'in_fodf.nii.gz' does not exist in the `cwd`, they are in the `get_home()`

Reply:
That's true. I coded this like so to make it possible to run the test in local on your machine. Now, to make it run on Jenkins, first, we have to upload the files on the google drive, then I'll add the Scilpy fetcher back (I know we want to get rid of it, let's make this in a separate process)